2018/10/16/src/main/java/me/lemire/microbenchmarks/algorithms/Utf8Validate.java

```diff
@@ -156,8 +156,10 @@ private static boolean isWellFormedSlowPath(byte[] bytes, int off, int end) {
   1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,  9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,9,
   7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,  7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,7,
   8,8,2,2,2,2,2,2,2,2,2,2,2,2,2,2,  2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,
-  10,3,3,3,3,3,3,3,3,3,3,3,3,4,3,3, 11,6,6,6,5,8,8,8,8,8,8,8,8,8,8,8,
+  10,3,3,3,3,3,3,3,3,3,3,3,3,4,3,3, 11,6,6,6,5,8,8,8,8,8,8,8,8,8,8,8
+};
```

Comment:
Split the tables in two. In principle this removes the need for a `+ 256` in the critical dependency chain, although in practice it probably gets folded into an offset in the addressing mode for the load instruction, so it may not matter. Seems cleaner though!